### PR TITLE
Document dynamic email settings in Watcher

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we document which of the email notification settings in
Watcher can be set dynamically via the cluster settings API.

### CODE REVIEW

File: docs/reference/settings/notification-settings.asciidoc [hunk 1]
```diff
@@ -93,59 +94,59 @@ can specify the following email account attributes:
   for the emails sent from the account. See {xpack-ref}/actions-email.html#email-action-attributes[
```
Comment (jakelandis) [inline docs/reference/settings/notification-settings.asciidoc]: `email_defaults` are also dynamic.

**Review by jakelandis — CHANGES_REQUESTED** [review]
thanks @danielmitterdorfer ! this looks good.  just missing `email_defaults`

**Comment by danielmitterdorfer** [comment]
@jakelandis thanks for the review. I've added a note to `email_defaults` as well now.

**Review by jakelandis — APPROVED** [review]

**Comment by jakelandis** [comment]
LGTM thanks!

**Comment by danielmitterdorfer** [comment]
Thanks for the review!

**Comment by danielmitterdorfer** [comment]
Backported to 6.x in https://github.com/elastic/elasticsearch/commit/0e6afcb79968e4953140a81d02fe271655c08bee.

**Pull request merged**